package.json

```diff
@@ -186,23 +152,23 @@
     "eslint-plugin-react-hooks": "^5.2.0",
     "globals": "^16.0.0",
     "husky": "^9.0.11",
-    "jest": "^29.5.0",
-    "jest-environment-jsdom": "^29.5.0",
     "lint-staged": "^15.0.0",
     "prettier": "^3.3.3",
     "react-test-renderer": "^18.2.0",
     "stylelint": "^16.11.0",
     "stylelint-config-prettier-scss": "^1.0.0",
     "stylelint-config-standard-scss": "^14.0.0",
+    "ts-jest": "^29.3.2",
```

Comment:
I am not sure why, but you added back `ts-jest` and `@types/jest`, I dont think its expected as this PR contains the switch to Vite?